Makefile

```diff
@@ -45,6 +45,16 @@ IMAGE_TAG=$(shell ./tools/image-tag)
 
 all: $(SCOPE_EXPORT)
 
+update-cri:
+	curl https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto > runtime/api.proto
```

Comment:
Should this also add a header to that file?

Reply:
A header? Not sure I understand what you meant here.

Reply:
Do we want it to add a big warning to the top to say something like "// DO NOT EDIT: This file is imported from kubernetes, update it with make blabla"